src/components/sound/index.tsx

```diff
@@ -59,9 +61,24 @@ export const SoundButton: React.FC<SoundButtonProps> = ({ sound }) => {
       }
     }
 
+    if (querySounds.length) {
+      decodeURIComponent(querySounds)
+        .split(';')
+        .forEach(item => {
+          const [id, volume] = item.split(',')
+          if (id === sound.id) {
+            initialState = {
+              id,
+              volume: parseFloat(volume),
+              active: true,
+              loaded: true
```

Comment:
You can not pass `loaded` as `true` by default, is better to pass as `false`, then keep `onCanPlay` handle this.

Reply:
Ok, thanks, I'll adjust this part